src/models/Studio.ts

```diff
@@ -88,16 +89,16 @@ export function start(filePath: string, port: number = DEFAULT_PORT): void {
         }
       } catch (e) {
         console.error(`Live Server: An invalid event has been received. See details:\n${event}`);
-      }      
+      }
     });
   });
-  
+
   wsServer.on('close', (socket: any) => {
     sockets.splice(sockets.findIndex(s => s === socket));
   });
 
   server.listen(port, () => {
-    const url = `http://localhost:${port}?liveServer=${port}`;
+    const url = `http://localhost:${port}?liveServer=${port}&_sv=${studioVersion}`;
```

Comment:
are you sure this is a solution? In linked issue nobody specified that `1.4.0` of CLI was used, and older version did not have latest studio


also where is this `_sv` param coming from?

Reply:
Yes, it's not the most exciting solution, but we should find a way to completely disable the browser cache in the local HTTP server we are using to start Studio.

The issue happens for new CLI versions, even though we have the latest studio version. 

`_sv` -> studio version.

Reply:
https://www.loom.com/share/0a41904d9537411f98b2b46bc25f0344?sid=acb7be86-6da2-47f1-a568-5f328fe9221c

Reply:
oh crap 🤔 

ok, so `_sv` has no meaning for studio really, it is just a hack to force "refresh". 


I think it is good, also useful as it informs what studio version you are using.

This is why, what about using full name for the query param? so it is immediately clear what that is?

Reply:
@derberg done, you take a look?